test/test_self_healing_scaling.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+source test/utils.sh
+
+function cleanup() {
+  kubectl delete namespace loadgen
+}
+
+trap cleanup EXIT
+
+# get keptn api details
+KEPTN_ENDPOINT=https://api.keptn.$(kubectl get cm keptn-domain -n keptn -ojsonpath={.data.app_domain})
+KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
+
+# test configuration
+PROJECT="sockshop"
+SERVICE="carts"
+STAGE="production"
+
+PROMETHEUS_SERVICE_VERSION=${UNLEASH_SERVICE_VERSION:-0.3.4}
+
+kubectl delete namespace sockshop-dev
+kubectl delete namespace sockshop-staging
+kubectl delete namespace sockshop-production
+keptn delete project sockshop
+
+keptn create project sockshop --shipyard=./test/assets/shipyard_self_healing_scale.yaml
+
+# Prerequisites
+rm -rf examples
+git clone --branch master https://github.com/keptn/examples --single-branch
+
+cd examples/onboarding-carts
+
+keptn onboard service carts --project=sockshop --chart=./carts
+keptn onboard service carts-db --project=sockshop --chart=./carts-db --deployment-strategy=direct
+keptn send event new-artifact --project=sockshop --service=carts-db --image=mongo
+
+keptn add-resource --project=sockshop --service=carts --stage=dev --resource=jmeter/basiccheck.jmx --resourceUri=jmeter/basiccheck.jmx
```

Comment:
Please add a comment stating what we are adding in which stage. Best would be to refer to our tutorial/docs.